sdk/scala-sdk/src/main/scala/com/akkaserverless/scalasdk/valueentity/ValueEntityOptions.scala

```diff
@@ -16,10 +16,13 @@
 
 package com.akkaserverless.scalasdk.valueentity
 
+import scala.collection.immutable
+
 import com.akkaserverless.javasdk.PassivationStrategy
 import com.akkaserverless.scalasdk.EntityOptions
 
 /** Root entity options for all value based entities. */
 trait ValueEntityOptions extends EntityOptions {
   def withPassivationStrategy(strategy: PassivationStrategy): ValueEntityOptions
+  def withForwardHeaders(headers: immutable.Set[String]): ValueEntityOptions
```

Comment:
Btw, there is no default impl for this one. I guess this will be added in the process of generating code for ValueEntity

Reply:
Is that not in ValueEntityOptionsImpl? (or maybe misunderstanding)

Reply:
(so maybe we need another adapter)